extensions/notebook/package.json

```diff
@@ -186,7 +186,7 @@
         }
       },
       {
-        "command": "notebook.command.searchUntitledBook",
+        "command": "notebook.command.searchProvidedBook",
```

Comment:
are there any places where we're referencing notebook.command.searchUntitledBook?

Reply:
we use it for search on the items in Provided Books viewlet.

Reply:
ok. just wanted to make sure that anything that referenced the old command name has been updated.